nodelet/src/nodelet.cpp

```diff
@@ -53,13 +52,29 @@
 #include "nodelet/NodeletLoad.h"
 #include "nodelet/NodeletUnload.h"
 
+#ifndef _WIN32
+#include <uuid/uuid.h>
+#else
+#include <rpc.h>
+#endif
```

Comment:
please indent the includes for readability (similar to https://github.com/ros/bond_core/pull/38/commits/5ebdaf7b79f060fa546d6f55c69a0e124cff92d5)

Reply:
proper idented